system/jlib/jlib.hpp

```diff
@@ -204,6 +204,8 @@ class PointerArrayOf : public PointerArray
     inline bool zap(TYPE * x)                   { return PointerArray::zap(x); }
 };
 
+enum SSLCfg { SSLNone = 0, SSLBoth, SSLOnly };
```

Comment:
I almost always do this for enums just as a habit so its clearly noted what the values are.